google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -17,7 +17,7 @@
 select.project.signin=Sign in with your Google account to list your Google Developers Console projects.
 
 cloud.project.label=Project\:
-
+appengine.flex.tools.menu.item.text=App Engine flex environment deployment
```

Comment:
I would be more precise with this label since we could have multiple menu items.
Also probably menu.tools since the 'tools' refers to a menu specific header.
And .text should be .label.